hayeon/programmers-0/숨어있는-숫자의-덧셈.js

```diff
@@ -0,0 +1,11 @@
+function solution(my_string) {
+  const regexArr = my_string.replace(/[^0-9]/g, " ").split(" ");
+
+  return regexArr.reduce((acc, cur) => (acc === "" ? acc : acc + +cur), 0);
+}
```

Comment:
```suggestion
function solution(my_string) {
  const regexArr = my_string.replace(/[^0-9]/g, " ").split(" ");

  return regexArr.reduce((acc, cur) => acc + Number(cur), 0);
}
```
- 풀이 잘봤습니다 👍 reduce할 때 삼항 연산자를 하신 이유가 있을까요?? 삼항 연산자 없이도 가능합니다!

Reply:
아무것도 없는 빈 문자열상태 일때 고려해서 삼항연산자를 했숩니다,, 윤우님 코드 참고하니 reduce할 때 빈문자열을 고려하지 않아도 됐던거군여,,😮